src/components/DateTimeField/index.jsx

```diff
@@ -21,7 +21,7 @@ class DateTimeField extends React.Component {
         : getDateWithDashes(this.props.input.value),
       time: (this.props.utcTimeZone
         ? getTimeStringUTC(this.props.input.value)
-        : getTimeString(this.props.input.value)) || '12:00',
+        : getTimeString(this.props.input.value)) || '12:00 AM',
```

Comment:
Does this AM work out of the box? Should it not be 00:00?

Reply:
<img width="633" alt="Screenshot 2024-10-11 at 3 12 19 PM" src="https://github.com/user-attachments/assets/cb673394-4972-48bc-87e6-c15a6e2a59d4">
Yes, you're right.
Updated